frontend/src/containers/AddWorkspace/styles.module.sass

```diff
@@ -0,0 +1,81 @@
+.AddWorkspace
+  font-family: 'Recursive'
+  width: 100%
+  height: 100vh
+  display: flex
+  justify-content: center
+  align-items: center
+  &__container
+    width: 1024px
+    height: 75vh
+    // background-color: red
+    border-radius: 6px
+    display: flex
+    box-shadow: 0 0 6px rgba(0, 0, 0, 0.2)
+    min-height: 535px
+    overflow: hidden
+
+.inputWorkspace
+  min-width: 40%
+  height: 100%
+  display: flex
+  flex-direction: column
+  justify-content: space-between
+  align-items: center
+
+.logoContainer
+  width: 100%
+  height: 75px
+  display: flex
+  align-items: center
+  &__item
+    height: 50px
+    margin-left: 10px
+
+.inputContainer
+  width: 80%
+  &__header
+    font-size: 22px
+    letter-spacing: 0px
+    line-height: 1.2
+    margin-bottom: 12px
+    font-weight: 600
+
+.inputButton
+  width: 130px
+  font-size: 22px
+  margin-bottom: 20px
+  background-color: #d1d0d0
```

Comment:
Colors should be imported from colors.ts

Reply:
Don't see this file